controller/array_action/config.py

```diff
@@ -8,3 +8,5 @@
 
 # volume context
 CONTEXT_POOL = "pool"
+
+UTF_8 = "utf-8"
```

Comment:
why did you resolve this? I still thinks this is the wrong location for it

Reply:
This is general constant. Not svc related. Even if it currently is used in svc.

Reply:
even if you want single-use consts here, the way I see this file - "config" means values that we want the option to easily change in the future, and since this value won't change in python 3, we won't need to re-"configure" it.

regardless of where you eventually place it, make sure you don't duplicate the same const (e.g. `utils.py`)

Reply:
I don't remember when we talked about it, but I don't think I should open a ticket for a small rename.
if you think it is too much for this ticket, please open a new one and add the number here (we shouldn't treat `config` as `constants` without at least a ticket for renaming it)

Reply:
Removed from config